handlers/Contact/components/NewsletterInfo/index.js

```diff
@@ -10,10 +10,10 @@ class NewsletterInfo extends React.Component {
       <section className="NewsletterInfo">
         <header className="NewsletterInfo-header">
           <span className="NewsletterInfo-header-small">Subscribe to the</span>
-          <span>Skookum Files</span>
+          <span>Skookum Gazette</span>
         </header>
-        <p className="NewsletterInfo-content">Where the bounds of literature are stretched, ideas are baked to a glorious temperature, and awesomeness is illustrated: the Skookum Newsletter wants to reach your eye balls and crawl into your imagination. We want to build things with you, we want to share extraordinary concepts, and we want to basically change the world.</p>
-        <p className="NewsletterInfo-content">Sign up. Be bold.</p>
+        <p className="NewsletterInfo-content">Where the bounds of literature are stretched, ideas are baked to a glorious temperature, and awesomeness is illustrated: the Skookum Gazette wants to reach your eye balls and crawl into your imagination. We want to build things with you, we want to share extrodinatary concepts, and we want to basically change the world.</p>
+        <p className="NewsletterInfo-content NewsletterInfo-content--uppercase">Subscribe. Be bold.</p>
```

Comment:
fwiw, I’m okay with throwing one off style changes like this inline rather than through a class. No change required.